src/containers/shared/components/SearchResults/SearchResults.tsx

```diff
@@ -0,0 +1,138 @@
+import { useEffect, useRef, useState } from 'react'
+import './SearchResults.scss'
+
+import { useTranslation } from 'react-i18next'
+import { useAnalytics } from '../../analytics'
+import { SearchResultRow } from './SearchResultRow'
+
+interface SearchResultsProps {
+  currentSearchValue: string
+  setCurrentSearchInput: (string) => void
+}
+
+const SearchResults = ({
+  currentSearchValue,
+  setCurrentSearchInput,
+}: SearchResultsProps): JSX.Element => {
+  const analytics = useAnalytics()
+  const { t } = useTranslation()
+  const xrplmetaWSRef = useRef<WebSocket | null>(null)
+  const [currentSearchResults, setCurrentSearchResults] = useState<any[]>([])
+  const [XRPUSDPrice, setXRPUSDPrice] = useState(0.0)
+
+  const connectXRPLMeta = () => {
+    xrplmetaWSRef.current = new WebSocket(
```

Comment:
I dont think websockets are the correct pattern for this.  This is for long running connections.  These arent really long running.